### PR TITLE
Batch requests to SSM API

### PR DESCRIPTION
**Pull request by ags**
As reported in https://github.com/remind101/ssm-env/issues/10, the SSM GetParameters API is limited to a maximum of ten parameter names in a single request. Attempting to fetch more than that fails.

This splits up the parameter names into batches of ten to avoid that limit.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -126,39 +122,71 @@ func (e *expander) expandEnviron(decrypt bool) error {
...
 	}
 
+	names := make([]string, len(uniqNames))
+	i := 0
+	for k := range uniqNames {
+		names[i] = k
+		i++
+	}
+
+	batchSize := 10
```
Comment (ejholmes) [inline main.go]: Can you expand the tests to include one that exercises batching? The implementation here looks correct, but it doesn't look like all the code paths are touched.

Comment (ags) [reply main.go]: No problem, I should have done that to begin with 😬 
I've updated it with a test that should test two batches of size one.

**Comment by matt-kg** [comment]
Seconding that. It would be nice if we can get an updated release.

**Comment by ejholmes** [comment]
Sorry for the delay on this one. I'll try to get a release cut with what's in master sometime this week.

**Pull request merged**